docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -759,6 +777,30 @@ Be careful if you have persistent volumes or other important resources you want
 {% endnavtab %}
 {% endnavtabs %}
 
+### Verify
```

Comment:
I've moved the "Verify" section immediately after the "Deploy" section since that would be the correct order-of-operations.